.github/workflows/push_pull_request_test.yml

```diff
@@ -0,0 +1,16 @@
+name: Automated testing workflow
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the stack
+        run: |
+          cd ./src/cc_catalog_airflow
+          cp env.template .env
+          docker-compose up -d
+      - name: Test
+        run: |
+          sleep 10
```

Comment:
This might introduce some flakiness in the tests. A more resilient approach would probably loop on `sleep`, based on the `docker container ls` response.